Documentation/docs/learn/python_quick_start.md

```diff
@@ -0,0 +1,495 @@
+preserving metadata:
+
+```py
+vtk_image = itk.vtk_image_from_image(image)
+
+image = itk.image_from_vtk_image(vtk_image)
+```
+
+### ITK and napari
+
+An `itk.Image` can be converted to and from a [napari.layers.Image](https://napari.org/stable/api/napari.layers.Image.html#napari.layers.Image) while
+preserving metadata with the [itk-napari-conversion package](https://github.com/InsightSoftwareConsortium/itk-napari-conversion).
+
+### ITK Python types
+
+| C++ type             | Python type          | NumPy dtype  |
+| -------------------- | -------------------- | ------------ |
+| float                | itk.F                | np.float32   |
+| double               | itk.D                | np.float64   |
+| unsigned char        | itk.UC               | np.uint8     |
+| std::complex\<float> | itk.complex\[itk.F\] | np.complex64 |
+
+This list is not exhaustive and is only presented to illustrate the type names. The complete list of types can be found in the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf).
+
+Types can also be obtained from their name in the C programming language:
+
+```py
+itk.F == itk.ctype('float') # True
+```
+
+To cast the pixel type of an image, use `.astype`:
+
+```py
+image = itk.imread(input_filename)
+
+# Cast to an unsigned char pixel type
+cast_image = image.astype(itk.UC)
+
+# Equivalent
+cast_image = image.astype(np.uint8)
+
+itk.imwrite(cast_image, output_filename)
+```
+
+### Metadata dictionary
+
+An `itk.Image` has a metadata dict of `key: value` pairs.
+
+The metadata dictionary can be retrieved with:
+
+```
+meta_dict = dict(image)
+```
+
+For example:
+
+```
+In [3]: dict(image)
+Out[3]:
+{'0008|0005': 'ISO IR 100',
+ '0008|0008': 'ORIGINAL\\PRIMARY\\AXIAL',
+ '0008|0016': '1.2.840.10008.5.1.4.1.1.2',
+ '0008|0018': '1.3.12.2.1107.5.8.99.484849.834848.79844848.2001082217554549',
+ '0008|0020': '20010822',
+```
+
+Individual dictionary items can be accessed or assigned:
+
+```
+print(image['0008|0008'])
+
+image['origin'] = [4.0, 2.0, 2.0]
+```
+
+In the Python dictionary interface to image metadata, keys for the spatial
+metadata, the *'origin'*, *'spacing'*, and *'direction'*, are reversed in
+order from `image.GetOrigin()`, `image.GetSpacing()`, `image.GetDirection()`
+to be consistent with the [NumPy array index order](https://scikit-image.org/docs/dev/user_guide/numpy_images.html#notes-on-the-order-of-array-dimensions)
+resulting from pixel buffer array views on the image.
+
+### Access pixel data with NumPy indexing
+
+Array views of an `itk.Image` provide a way to set and get pixel values with NumPy indexing syntax, e.g.:
+
+```
+In [6]: image[0,:2,4] = [5,5]
+
+In [7]: image[0,:4,4:6]
+Out[7]:
+NDArrayITKBase([[    5,  -997],
+                [    5, -1003],
+                [ -993,  -999],
+                [ -996,  -994]], dtype=int16)
+```
+
+### Input/Output (IO)
```

Comment:
IO vs I/O. The latter has been used in https://github.com/InsightSoftwareConsortium/ITK/pull/4320/files#diff-e3224d6eebdca0df59ba3fa95a671a4556289f04e5c0c00b6e41fdb7f8d5b7baR37